javaagent/build.gradle.kts

```diff
@@ -271,6 +273,36 @@ with(components["java"] as AdhocComponentWithVariants) {
   }
 }
 
+spdxSbom {
+  targets {
+    // Create a target to match the published jar name.
+    // This is used for the task name (spdxSbomFor<SbomName>)
+    // and output file (<sbomName>.spdx.json).
+    create("opentelemetry-java_opentelemetry-javaagent") {
+      configurations.set(listOf("baseJavaagentLibs"))
+      scm {
+        uri.set("https://github.com/" + System.getenv("GITHUB_REPOSITORY"))
+        revision.set(System.getenv("GITHUB_SHA"))
+      }
+      document {
+        name.set("opentelemetry-java_opentelemetry-javaagent")
+        namespace.set("https://opentelemetry.io/spdx/" + UUID.randomUUID())
```

Comment:
what's the significance of the name and namespace?

Reply:
https://github.com/open-telemetry/opentelemetry-java/pull/5948#discussion_r1537894386

> Per the documentation, the intent is to "provide an unambiguous mechanism for other SPDX documents to reference SPDX elements within this SPDX document".
and
"The URI does not have to be accessible. It is only intended to provide a unique ID"
https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#652-intent